lib/camerata.rb

```diff
@@ -0,0 +1,275 @@
+# frozen_string_literal: true
+require "camerata/version"
+require "camerata/parameters"
+require "camerata/app_versions"
+require "camerata/secrets"
+require 'thor'
+require 'erb'
+require 'json'
+require 'active_support'
+require 'yaml'
+require 'byebug'
+
+# rubocop:disable Metrics/ClassLength
+module Camerata
+  class Error < StandardError; end
+  class CLI < Thor
+    include Thor::Actions
+
+    def self.source_root
+      File.join(File.dirname(__FILE__), '..', 'templates')
+    end
+
+    def self.exit_on_failure?
+      true
+    end
+
+    method_option :without, default: '', type: :string, aliases: '-no'
+    desc "up", "starts docker-compose with orphan removal, defaults to blacklight"
+    def up(*args)
+      ensure_env
+      output = default_options(args, ['--remove-orphans'])
+      run_with_exit("#{docker_compose} up #{output.join(' ')}")
+    end
+
+    desc "stop", "stops the specified running service, defaults to all"
+    def stop(*args)
+      ensure_env
+      run("#{docker_compose} stop #{args.join(' ')}")
+      run_with_exit("rm -rf tmp/pids/*")
+    end
+
+    desc "restart", "restarts the specified running service, defaults to all"
+    def restart(*args)
+      ensure_env
+      run_with_exti("#{docker_compose} restart #{args.join(' ')}")
```

Comment:
"run_with_exit" yeah?

Reply:
totally. can you guess which method I added very very last after I'd done most of my testing? thank you for catching that